wagtail_footnotes_test_app/factories.py

```diff
@@ -1,9 +1,12 @@
 import factory
 import wagtail_factories
+import uuid
```

Comment:
fly-by comment: worth moving the test app in `tests/`
then you can configure the packager to ignore the whole of `tests/` for smaller packages